packages/cadl-rlc-emitter/test/integration/generated/clients/interfaces/typescript.json

```diff
@@ -8,5 +8,6 @@
     "version": "1.0.0"
   },
   "isCadlTest": true,
-  "title": "InterfaceClient"
+  "title": "InterfaceClient",
+  "generateTest": true
```

Comment:
wondering why are we adding generate test for all these integration cadl ranch test ?

Reply:
No we only enable part of them not all. But I think we could limit to one folder if unnecessary.